sample/ios/Podfile.lock

```diff
@@ -455,7 +455,7 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
-  CocoaAsyncSocket: 694058e7c0ed05a9e217d1b3c7ded962f4180845
+  CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
```

Comment:
wondering why this has changed

Reply:
When I bump `sentry-cocoa` I have to delete the whole Podfile.lock because otherwise pod install doesn't work. At this point maybe we should consider stopping committing lock files?

Reply:
yeah its also from the sample, so not important, I'd remove, we do that on dart/flutter https://github.com/getsentry/sentry-dart/blob/main/.gitignore#L22-L23